City-of-Helsinki/drupal-helfi-rekry
# UHF-6565: Fix organization description

# [UHF-6565](https://helsinkisolutionoffice.atlassian.net/browse/UHF-6565)


## What was done
* Moved template file from custom module to custom theme.
* Fixed getting organization description. Previously, it was not possible to get the term description in the case where
the term image was missing.

## How to install

* Make sure your instance is up and running on correct branch.
  * `git pull origin UHF-6565-fix-organization-description`
  * `make fresh`
* Run `make drush-cr`

## How to test
* Go to the content listing and open some job listings.
* Make sure the right sidebar shows the image and description text correctly:
  * Image:
    * If the job listing has an image, it's shown.
    * If not, the organization term image is shown, if available.
    * If not, the placeholder is shown.
  * Description:
    * If the job listing has an organization description, it's shown.
    * If not, the organization term description is shown, if available.

## Designers review
